packages/cli/src/integration-ref.test.ts

```diff
@@ -0,0 +1,115 @@
+import { describe } from 'node:test'
+import { test, expect } from 'vitest'
+import { formatIntegrationRef, IntegrationRef, parseIntegrationRef } from './integration-ref'
+
+const path = '/my/path'
+const uuid = 'a0eebc99-9c0b-4ef8-bb6d-6bb9bd380a11'
+const name = 'myintegration'
+
+describe('parseIntegrationRef', () => {
+  test('parse empty string should return undefined', () => {
+    // arrange
```

Comment:
I also like the AAA commenting pattern for tests, we should it more often for complex/long tests 🤩